go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/portainer/compose-unpacker
 
-go 1.18
+go 1.17
```

Comment:
I think we recently switched to using go 1.18 - any reason why you want to downgrade the go module version?